src/core/prototypes/plot_segments/axis.ts

```diff
@@ -744,16 +753,29 @@ export class AxisRenderer {
         }
       }
     }
+
+    if (offset) {
+      g.transform = {
+        x: offset.xOffset,
+        y: offset.yOffset,
+        angle: 0,
+      };
+    }
     return g;
   }
 
-  public renderCartesian(x: number, y: number, axis: AxisMode): Group {
+  public renderCartesian(
```

Comment:
Should there be an offset for Polar Coordinates as well?

In the `renderPolar` case the offset would be in the radial axis. So I guess for the data model and UI there would be another `radialOffset` value in the AxisOffset class?

Or is adding offset for polar coordinates out of scope?